Source/Classes/PINRemoteImageManager.m

```diff
@@ -116,6 +116,7 @@ @interface PINRemoteImageManager () <PINURLSessionManagerDelegate>
 @property (nonatomic, copy) id<PINRequestRetryStrategy> (^retryStrategyCreationBlock)(void);
 @property (nonatomic, copy) PINRemoteImageManagerRequestConfigurationHandler requestConfigurationHandler;
 @property (nonatomic, strong) NSMutableDictionary <NSString *, NSString *> *httpHeaderFields;
+@property(nonatomic, assign) BOOL isTtlCache;
```

Comment:
To match the codebase, space before ( -- which is not done in some other codebases.

I would suggest a different name, maybe respectCacheControlHeaders, trackImageExpiry, enforceCacheAgeLimits - since this class is not by itself a cache, it's more like a class managing a cache than "isTtlCache".

Reply:
Spacing added.
Since the underlying cache is not necessarily PINCache, this var is storing (caching) whether the cache implements the optional `- (void)setObjectOnDisk:(id)object forKey:(NSString *)key withAgeLimit:(NSTimeInterval)ageLimit` method, that is whether it is a Ttl cache or not. So I think the name is accurate, (although maybe could be `cacheIsTtlCache`)...